classes/04class/exercises/c04-iac02/carineoliveira/_provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "aws" {
+  region     = "ap-southeast-2"
+  access_key = "AKIAJRJOGK3AJN6JZ3CA"
+  secret_key = "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"

```

Comment:
remember to not include keys in the files you push to the repository.

Reply:
yep, I forgot to remove before submitting the PR, however, I deleted it from my AWS account already. AWS sent a notification about that, very smart!